src/main/java/com/browserstack/automate/ci/jenkins/qualityDashboard/QualityDashboardInit.java

```diff
@@ -105,41 +111,47 @@ private static boolean initialQDSetupRequired(BrowserStackCredentials browserSta
         return false;
     }
 
-    private static List<String> getAllPipelines(BrowserStackCredentials browserStackCredentials) {
-        List<String> allPipelines = new ArrayList<>();
+    private static List<PipelineInfo> getAllPipelines(BrowserStackCredentials browserStackCredentials) {
+        List<PipelineInfo> allPipelines = new ArrayList<>();
         Jenkins jenkins = Jenkins.getInstanceOrNull();
         Integer totalPipelines = 0;
 
         if (jenkins != null) {
             totalPipelines = jenkins.getAllItems().size();
             jenkins.getAllItems().forEach(job -> {
                 try {
+                    String itemType = QualityDashboardUtil.getItemTypeModified(job);
+                    boolean isWorkflowJob = job instanceof WorkflowJob;
+                    
                     // Logging job details
                     apiUtil.logToQD(
                         browserStackCredentials,
                         String.format(
                             "Job name: %s, instance type: %s, and is_workflow_job: %s",
                             job.getName(),
-                            job.getClass().getSimpleName(),
-                            (job instanceof WorkflowJob) ? "yes" : "no"
+                            itemType,
+                            isWorkflowJob ? "yes" : "no"
                         )
                     );
+                    if (itemType != null) {
+                        String pipelineName = job.getFullName();
+                        allPipelines.add(new PipelineInfo(pipelineName, itemType));
+                    }
+                    else{
+                        apiUtil.logToQD(browserStackCredentials, "Skipping job: " + job.getName() + " as it is not a Job or Folder instance");
+                    }
+                    
                 } catch (JsonProcessingException e) {
                     // Handling the exception and logging an error
-                    System.err.println("Error processing JSON for job: " + job.getName());
+                    LOGGER.info("Error processing JSON for job: " + job.getName() + " - " + e.getMessage());
```

Comment:
These should be logger error right? And till will also be displayed to user correct?